requirements.in

```diff
@@ -0,0 +1,32 @@
+--use-feature="2020-resolver"
```

Comment:
Grand yup, we can drop this once pip 20.3 is released, due October 2020.

https://pip.pypa.io/en/stable/news/#id26